python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -648,6 +648,21 @@ def main():
     else:
         print 'stdlib ssl %s' % (OPENSSL_VERSION,)
 
+    try:
+        # Import OpenSSL first to suppress cryptography warnings effectively.
+        import OpenSSL
+        from OpenSSL import SSL, crypto, rand, __version__ as pyopenssl_version
```

Comment:
```suggestion
        from OpenSSL import crypto, SSL, rand, __version__ as pyopenssl_version
```

just a suggestion... maybe this is enough

Reply:
Good catch! But this not needed any more when filtering exact messaging, so I'm going to revert the change.